scipy/optimize/_differentialevolution.py

```diff
@@ -309,8 +372,14 @@ def __init__(self, func, bounds, args=(),
                  strategy='best1bin', maxiter=None, popsize=15,
                  tol=0.01, mutation=(0.5, 1), recombination=0.7, seed=None,
                  maxfun=None, callback=None, disp=False, polish=True,
-                 init='latinhypercube'):
-
+                 init='latinhypercube', pool=None):
+        
+        if pool is None:
+            self.pool = SPool()
```

Comment:
Fixed in https://github.com/pavelponomarev/scipy/commit/0b820228cbe2d2306f5d6865cab5df9deac41456